### DISCUSSION TITLE
The Radical Uselessness Contest [jiarong zhou and xiaojing xiong]

### DISCUSSION THREAD

**Discussion by kulo-b** [realitydeslab/playshop-summer-2024#24] [Proposal]
**Research Question**：When everything is replaced by machines, what exactly are we left with? When every basic human function is outsourced to a robot, what kind of change will come to human subjectivity?
**研究问题**：当一切都被机器取代，我们究竟还剩下什么？当人类的每一个基本功能都外包给机器人，人类的主体身份将迎来什么样的改变？

**key words**
人工智能（Artificial Intelligence）
- 主体性危机（Crisis of Subjectivity）
- 自我认同异化（Alienation of Self-Identity）
- 人类角色（Human Role）
- 主体性放弃（Abandonment of Subjectivity）

**Artefact / Media:**
线上模拟器 Online simulator

**Abstract:**

The "Most Useless Human Contest" is a unique talent show organized in the context of the postmodern era, which aims to find those humans who, in a society with a high degree of artificial intelligence, appear to be the most useless, make almost zero contribution to society, and even lack the ability to take care of themselves in basic life. This contest is an ironic reflection of the crisis of human subjectivity, which reveals that under the impact of mechanical intelligence, human self-identity is undergoing a kind of alienation.

In this era, artificial intelligence not only possesses memory, computing power and knowledge base beyond human beings, but also gradually replaces many roles of human beings in work and life. Humans have begun to change from the dominant player in life to a passive bystander, and even become the "subordinate" of AI. In this process, the "uselessness" of human beings has become a common phenomenon in society.

The "Most Useless Human Contest" pushes this "uselessness" to the extreme through the exaggerated form of the talent show, which is not only a profound reflection on the role of human beings in modern society, but also a critique of the negative impacts that may be brought about by technological progress. The performance of the contestants in the competition, although a farce on the surface, is in essence a warning for the future of humanity: if human beings continue to give up their subjectivity and agency, they may eventually become truly "useless".

"最没用人类大赛"是一场在后现代时期背景下举办的独特选秀活动，它的宗旨是寻找那些在人工智能高度发展的社会中，显得最无用、对社会的贡献几乎为零，甚至缺乏基本生活自理能力的人类。这个大赛是对人类主体性危机的一种讽刺性反映，它揭示了在机械智能的冲击下，人类的自我认同正在经历一种异化。

在这个时代，人工智能不仅拥有超乎人类的记忆、计算能力和知识储备，还逐渐取代了人类在工作和生活中的许多角色。人类开始从生活的主导者变成被动的旁观者，甚至变成了人工智能的“附庸”。在这个过程中，人类的“无用化”成为了社会的一种普遍现象。

"最没用人类大赛"通过夸张的选秀形式，将这种“无用化”推向了极致，它不仅是对现代社会中人类角色的深刻反思，也是对技术进步可能带来的负面影响的批判。大赛中的参赛者们的表现，虽然在表面上是一场闹剧，但实质上却是对人类未来的一种警示：如果人类继续放弃自己的主体性和能动性，最终可能会变得真正“无用”。

**Visual Reference / Illustrations**
![QQ图片20240626004051](https://github.com/realitydeslab/playshop-summer-2024/assets/173977885/4f662c7b-3f0a-44f4-8d65-a33654420405)
![23fc5209c991c4d2c590e1c098b3f76](https://github.com/realitydeslab/playshop-summer-2024/assets/173977885/c50d459d-4b30-4ff1-87c7-2d81de407764)

**References:**
[主体交流异化与现代性迷失.pdf](https://github.com/user-attachments/files/16007754/default.pdf)
[智能时代身体主体性的颠覆与重构%0a——兼论人类与人工智能的主体间性.pdf](https://github.com/user-attachments/files/16008073/0a.pdf)

**Comment by kulo-b** [comment]
https://www.pewresearch.org/internet/2018/12/10/artificial-intelligence-and-the-future-of-humans/

**Comment by glitch-kraft** [comment]
https://www.youtube.com/watch?v=Mc5Y3g4J5Yg

**Reply by glitch-kraft** [reply]
https://www.vice.com/en/article/vdq4y9/doing-nothing-has-become-a-sport-in-south-korea

**Comment by glitch-kraft** [comment]
https://www.byobworldwide.com/

**Comment by kulo-b** [comment]
Everyone can participate in this nonsense imaginary world event
**The rules :**
  Rule 1：
       - The server is like a VRchat, where users can observe each other in the same vr space together with their vr devices.
  Rule 2 ：
       -The server has an editable timeline, just like a person can write whatever they want on any page of a notepad. Users can put down whatever they want at any point in the timeline.

In the end, these edited "notepads" will be filled with people's scattered and unorganized thoughts and ideas, and eventually these notepad worlds will become a strange three-dimensional movie, or some non-sence meme.

大家可以参与这一废话想象世界活动
**规则如下**
规则1：
     这一服务器像VRchat， 用户可以通过vr设备共同相处在同一个vr空间之中相互观测
规则2 ：
      
服务器拥有可编辑的时间轴，就像人可以在一本记事本的任何一页写下他们想写的东西一样。用户可以在时间轴的任何一个时间点放下他们想放的东西。

最后，这些被编辑完成的“记事本”，将布满人们各自零散的、无序的事物与想法，最终这些记事本世界将或许会成为一个奇怪的三维影片，或者一些non-sence meme。
![QQ图片20240628155436](https://github.com/realitydeslab/playshop-summer-2024/assets/173977885/ff662d26-229e-41b8-b7be-0d8b895894dc)
![QQ图片20240628160208](https://github.com/realitydeslab/playshop-summer-2024/assets/173977885/000de7b4-45dd-4356-a5b7-7660a40a496c)

**Comment by glitch-kraft** [comment]
https://www.dailymotion.com/video/x49ttbr

**Comment by botaohu** [comment]
https://glif.app/glifs

**Comment by kulo-b** [comment]
![关键帧](https://github.com/realitydeslab/playshop-summer-2024/assets/173977885/05f10b1a-6ab5-4979-b045-371df54d6f29)

标语：When all the work on Earth is done by robots, humans are keen to compete for the title of "Most Useless Human".

**Comment by kulo-b** [comment]
Let’s have a “Most Useless Human contest”。just To run for that most useless person, to empty yourself, to laze around, and to do nothing. Re-gaz yourself in the meditation of emptiness, discover yourself. Put yourself in your first place.

**Comment by glitch-kraft** [comment]
**Title**

The Radical Uselessness Contest

**Tagline**
Run for the most useless one, laze around and do literally nothing. Re-gaze and discover yourself through the meditation of futility. Compete for the sake of absurdity of competing.
